tests/rkt_net_test.go

```diff
@@ -838,3 +845,113 @@ func NewTestNetLongName() testutils.Test {
 		testNetCustomNatConnectivity(t, nt)
 	})
 }
+
+func mockFlannelNetwork(t *testing.T, ctx *testutils.RktRunCtx) (string, networkTemplateT, error) {
+	// write fake flannel info
+	subnetPath := filepath.Join(ctx.DataDir(), "subnet.env")
+	file, err := os.Create(subnetPath)
+	if err != nil {
+		return "", networkTemplateT{}, err
+	}
+	mockedFlannel := fmt.Sprintf("FLANNEL_NETWORK=10.1.0.0/16\nFLANNEL_SUBNET=10.1.17.1/24\nFLANNEL_MTU=1472\nFLANNEL_IPMASQ=true\n")
```

Comment:
Look at https://github.com/coreos/rkt/blob/701c0f07b9e03cc7cced48364aa370ad3ba35bed/stage1/init/init.go#L252 and then usage of this in https://github.com/coreos/rkt/blob/701c0f07b9e03cc7cced48364aa370ad3ba35bed/stage1/init/init.go#L286 (justr replace " " with "\n")
I know that this looks as a overkill, but still - imo it's more readable.
Remember that code is ten times more read, then written.